examples/kafka/020-KafkaNodePool-controller-console-nodepool.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaNodePool
+metadata:
+  name: console-nodepool
+  labels:
+    strimzi.io/cluster: console-kafka
```

Comment:
Since we have two node pools to be used together, let's call this one `console-brokers` and the other one `console-controllers` (or similar). They will need to have distinct names anyway.